xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -4232,17 +4232,7 @@ glusterd_get_volumes(rpcsvc_request_t *req, dict_t *dict, int32_t flags)
         ret = 0;
         goto respond;
     }
-    if (flags == GF_CLI_GET_VOLUME_ALL) {
```

Comment:
Is this command not needed anymore ?

Reply:
It is never referenced from any place. Right, from when it was implemented I couldn't find any code using it.

Reply:
Yes, you are right. It hasn't been used for a long time...